hazelcast-client/src/main/java/com/hazelcast/client/cache/impl/ClientCacheProxy.java

```diff
@@ -403,8 +401,7 @@ protected void updateCacheListenerConfigOnOtherNodes(CacheEntryListenerConfigura
                 final ClientMessage request =
                         CacheListenerRegistrationCodec.encodeRequest(nameWithPrefix, configData, isRegister, address);
                 final ClientInvocation invocation = new ClientInvocation(client, request, address);
-                final Future future = invocation.invoke();
-                futures.add(future);
+                invocation.invoke();
```

Comment:
Okay, there was code after this, which used the futures:
```
        //make sure all configs are created
        //TODO do we need this ???s
        //        try {
        //            FutureUtil.waitWithDeadline(futures, CacheProxyUtil.AWAIT_COMPLETION_TIMEOUT_SECONDS, TimeUnit.SECONDS);
        //        } catch (TimeoutException e) {
        //            logger.warning(e);
        //        }
```
This commented out version was removed on 2015/10/05 by @serkan-ozal in commit 90a1a4f5d0d93b3c367e81f6f83e599e3983ce11
> Client side cache statistics implementation

So I assume it's safe to delete those unused futures, but could you please confirm this, Serkan?